robottelo/ui/audit.py

```diff
@@ -0,0 +1,49 @@
+# -*- encoding: utf-8 -*-
+"""Implements Audit UI"""
+
+from robottelo.ui.base import Base
+from robottelo.ui.locators import common_locators, locators
+from robottelo.ui.navigator import Navigator
+
+
+class Audit(Base):
+    """Manipulates Audit from UI"""
+
+    def navigate_to_entity(self):
+        """Navigate to Audit entity page"""
+        Navigator(self.browser).go_to_audits()
+
+    def _search_locator(self):
+        """There is no locator in Audit that can be directly associated with
+        search result
```

Comment:
Pls explain your point. There's a search bar, so searching can be performed (why `def filter`?). There's an entries list which is not our typical entity table of course, but it's still the entry we're searching for.
At the first glance, we could implement custom search here, which would return that last entry, couldn't we?

Reply:
nope, we can't. Open UI and try by youself. Basically, you cannot perform search by entity name or things like that, so it is a filter, not a search